node-src/tasks/gitInfo.ts

```diff
@@ -59,13 +60,21 @@ export const setGitInfo = async (ctx: Context, task: Task) => {
     patchBaseRef,
     fromCI: ci,
     interactive,
+    isLocalBuild,
   } = ctx.options;
 
+  const commitAndBranchInfo = await getCommitAndBranch(ctx, { branchName, patchBaseRef, ci });
+
   ctx.git = {
     version: await getVersion(),
-    ...(await getCommitAndBranch(ctx, { branchName, patchBaseRef, ci })),
+    gitUserEmail: await getUserEmail(),
```

Comment:
It seems this git command fails when there is no email configured, which is likely in a CI environment. 
```suggestion
    gitUserEmail: await getUserEmail().catch((e) => {
      ctx.log.debug('Failed to retrieve Git user email', e);
      return null;
    }),
```
I used this same pattern when [retrieving the uncommitted hash](https://github.com/chromaui/chromatic-cli/pull/798/files).